FAQ.md

```diff
@@ -13,6 +13,7 @@
 - [11. HTTP requests sent with Postman/curl/etc. hang if I'm debugging a test case with VSCode, why?](#11-http-requests-sent-with-postmancurletc-hang-if-im-debugging-a-test-case-with-vscode-why)
 - [12. How do I run test cases with my own ledger images?](#12-how-do-i-run-test-cases-with-my-own-ledger-images)
 - [13. How do I re-run a single job in the CI through the GitHub UI?](#13-how-do-i-re-run-a-single-job-in-the-ci-through-the-github-ui)
+- [14. How to fix the script configure, stuck at resolving npm registry (ubuntu)?](#14-how-to-fix-the-script-configure-stuck-at-resolving-npm-registry-ubuntu)
```

Comment:
@jagpreetsinghsasan I recommend being slightly more specific and mentioning IPv6 or the exact error message